<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYCurveDock</class>
 <widget class="QWidget" name="XYCurveDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>776</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabGeneral">
      <attribute name="title">
       <string>General</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabLine">
      <attribute name="title">
       <string>Line</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="2">
        <widget class="KComboBox" name="cbLineType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lLineType">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="2">
        <widget class="QCheckBox" name="chkLineIncreasingXOnly">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSpinBox" name="sbLineInterpolationPointsCount">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>intermediate points to evaluate between each two points</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>32</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lDropLines">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Drop lines</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lLineInterpolationPointsCount">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>Interm. points:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lLineSkipGaps">
         <property name="text">
          <string>Ignore Gaps:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lValuesGeneral_2">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Connection</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lLineIncreasingXOnly">
         <property name="text">
          <string>Monoton incr. X:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QCheckBox" name="chkLineSkipGaps">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_12">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSymbol">
      <attribute name="title">
       <string>Symbol</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabValues">
      <attribute name="title">
       <string>Values</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="0">
        <widget class="QLabel" name="lValuesColumn">
         <property name="text">
          <string>Column:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="lValuesPrecision">
         <property name="text">
          <string>Precision:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QSpinBox" name="sbValuesPrecision">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="QLineEdit" name="leValuesSuffix">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="13" column="0">
        <widget class="QLabel" name="lValuesText">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Text</string>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="lValuesSuffix">
         <property name="text">
          <string>Suffix:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="lValuesOpacity">
         <property name="text">
          <string>Opacity:</string>
         </property>
        </widget>
       </item>
       <item row="16" column="0">
        <widget class="QLabel" name="lValuesFont">
         <property name="text">
          <string>Font:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lValuesGeneral">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>General</string>
         </property>
        </widget>
       </item>
       <item row="16" column="2">
        <widget class="KFontRequester" name="kfrValuesFont"/>
       </item>
       <item row="17" column="2">
        <widget class="KColorButton" name="kcbValuesColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QSpinBox" name="sbValuesOpacity">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>The opacity ranges from 0 to 100, where 0 is fully transparent and 100 is fully opaque.</string>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lValuesDistance">
         <property name="text">
          <string>Distance:</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="KComboBox" name="cbValuesNumericFormat"/>
       </item>
       <item row="1" column="2">
        <widget class="KComboBox" name="cbValuesType">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="17" column="0">
        <widget class="QLabel" name="lValuesColor">
         <property name="text">
          <string>Color:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="KComboBox" name="cbValuesPosition"/>
       </item>
       <item row="14" column="2">
        <widget class="QLineEdit" name="leValuesPrefix">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lValuesRotation">
         <property name="text">
          <string>Rotation:</string>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="lValuesPrefix">
         <property name="text">
          <string>Prefix:</string>
         </property>
        </widget>
       </item>
       <item row="18" column="0">
        <spacer name="verticalSpacer_8">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>28</width>
           <height>47</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="0" colspan="2">
        <spacer name="verticalSpacer_7">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>86</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lValuesFormat">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Format</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSpinBox" name="sbValuesRotation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="suffix">
          <string>Â°</string>
         </property>
         <property name="minimum">
          <number>-360</number>
         </property>
         <property name="maximum">
          <number>360</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lValuesPosition">
         <property name="text">
          <string>Position:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="lValuesNumericFormat">
         <property name="text">
          <string>Numeric:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="2">
        <spacer name="verticalSpacer_6">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>86</width>
           <height>18</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="11" column="0">
        <widget class="QLabel" name="lValuesDateTimeFormat">
         <property name="text">
          <string>DateTime:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QComboBox" name="cbValuesDateTimeFormat"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lValuesType">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="NumberSpinBox" name="sbValuesDistance">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="suffix">
          <string> pt</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabAreaFilling">
      <attribute name="title">
       <string>Filling</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabErrorBars">
      <attribute name="title">
       <string>Error Bars</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
     <widget class="QWidget" name="tabMarginalPlots">
      <attribute name="title">
       <string>Margins</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="2">
        <widget class="QCheckBox" name="chkRugEnabled">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lRugEnabled">
         <property name="text">
          <string>Enabled:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>10</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lRugLength">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lRugOffset">
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lRugWidth">
         <property name="text">
          <string>Width:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lRugOrientation">
         <property name="text">
          <string>Orientation:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="lRug">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Rug Plot</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QComboBox" name="cbRugOrientation"/>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer_13">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>600</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2">
        <widget class="NumberSpinBox" name="sbRugLength">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="NumberSpinBox" name="sbRugWidth">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="minimum">
          <double>0.000000000000000</double>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="NumberSpinBox" name="sbRugOffset">
         <property name="toolTip">
          <string/>
         </property>
         <property name="whatsThis">
          <string>Spinbox for numbers.</string>
         </property>
         <property name="feedback" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KColorButton</class>
   <extends>QPushButton</extends>
   <header>kcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KFontRequester</class>
   <extends>QWidget</extends>
   <header>kfontrequester.h</header>
  </customwidget>
  <customwidget>
   <class>NumberSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>NumberSpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
